_format/1.1/index.md

```diff
@@ -1237,7 +1237,8 @@ section of the [compound document].
 
 The value of the `include` parameter **MUST** be a comma-separated (U+002C
 COMMA, ",") list of relationship paths. A relationship path is a dot-separated
-(U+002E FULL-STOP, ".") list of [relationship][relationships] names.
+(U+002E FULL-STOP, ".") list of [relationship][relationships] names. An empty
+value indicates that no related resource should be returned.
```

Comment:
```suggestion
value indicates that no related resources should be returned.
```

This is a really minor nit, but I think this should be plural.

Reply:
Applied the suggestion. It included some markup, which is not shown in GitHub preview. :disappointed:  f4fcfad1b8671ab8f9742ea2f84558670d6d11db Removed that one in 9b90b6380964d308d4079c9b3c2f126f8e94c304. Should be good to merge now.